designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/DBFetchingProcessRepository.scala

```diff
@@ -216,31 +222,31 @@ abstract class DBFetchingProcessRepository[F[_]: Monad](val dbRef: DbRef, action
       lastAction = lastActionData,
       lastStateAction = lastStateActionData,
       lastDeployedAction = lastDeployedActionData,
-      tags = tags.map(_.name).toList,
+      tags = tags.map(_.map(_.name).toList),
```

Comment:
I think it's easer to handle List than Option[List]

Reply:
Not fetched tabs can't be presented as an empty list of tabs :) Take a look that we already skipped fetching tags and process versions for list of processes. It was an easy way to bugs like showing and suddenly disappearing some information

Reply:
Ok, right now I understand why it's Option -> but still, did you check that this change doesn't break something on FE?